pyanaconda/kickstart.py

```diff
@@ -682,6 +679,8 @@ def ksshowwarning(message, category, filename, lineno, file=None, line=None):
                 if not report.is_valid():
                     message = "\n\n".join(map(str, report.error_messages))
                     raise KickstartError(message)
+                if strict_mode:
+                    kswarnings += report.warning_messages
```

Comment:
I am afraid that warnings from the DBus modules are not going to be logged in a non-strict mode.

Reply:
I would consider to "re-issue" the remote warnings here to make sure that they are processed the same way as the local warnings.

Reply:
Rewritten to re-issue these warnings.